src/WinRT.Runtime/Marshalers.cs

```diff
@@ -1672,12 +1672,24 @@ public static T FromAbi<T>(IntPtr nativeDelegate)
 
     internal static class Marshaler
     {
-        internal static Action<object> EmptyFunc = (object box) => { };
-        internal static Func<object, object> ReturnParameterFunc = (object box) => box;
-        internal static unsafe Action<object, IntPtr> CopyIntEnumFunc = 
-            (object value, IntPtr dest) => *(int*)dest.ToPointer() = (int)Convert.ChangeType(value, typeof(int));
-        internal static unsafe Action<object, IntPtr> CopyUIntEnumFunc =
-            (object value, IntPtr dest) => *(uint*)dest.ToPointer() = (uint)Convert.ChangeType(value, typeof(uint));
+        internal static readonly Action<object> EmptyFunc = Empty;

```

Comment:
I think this changed in one of your other PRs, so may need to rebase with those changes merged.